source/administration/mmctl-cli-tool.rst

```diff
@@ -4735,160 +4982,205 @@ mmctl user deleteall
 
 .. code-block:: sh
 
-     --confirm   Confirm you really want to delete the user and a DB backup has been performed
-     -h, --help  help for delete
+   --confirm   Confirm you really want to delete the user and a DB backup has been performed
+   -h, --help  help for delete
 
 **Options inherited from parent commands**
 
 .. code-block:: sh
 
-   --format string               the format of the command output [plain, json] (default "plain")
-   --insecure-sha1-intermediate  allows the use of insecure TLS protocols, such as SHA-1
-   --local                       allows communicating with the server through a unix socket
-   --strict                      will only run commands if the mmctl version matches the server one
+   --config-path string           path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")
+   --format string                the format of the command output [plain, json] (default "plain")
+   --insecure-sha1-intermediate   allows to use insecure TLS protocols, such as SHA-1
+   --insecure-tls-version         allows to use TLS versions 1.0 and 1.1
+   --local                        allows communicating with the server through a unix socket
+   --strict                       will only run commands if the mmctl version matches the server one
 
 mmctl user demote
 ^^^^^^^^^^^^^^^^^
 
 **Description**
 
-Demote a user to guest.
+Convert a user into a guest.
 
 **Format**
 
 .. code-block:: sh
 
-   mmctl user demote [users] [flags]
+  mmctl user demote [users] [flags]
 
 **Examples**
 
 .. code-block:: sh
 
-   user demote user1 user2
+  user demote user1 user2  
 
 **Options**
 
 .. code-block:: sh
 
-   -h, --help   help for demote
+  -h, --help   help for demote
 
 **Options inherited from parent commands**
 
+   --config-path string           path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")
+   --format string                the format of the command output [plain, json] (default "plain")
+   --insecure-sha1-intermediate   allows to use insecure TLS protocols, such as SHA-1
+   --insecure-tls-version         allows to use TLS versions 1.0 and 1.1
+   --local                        allows communicating with the server through a unix socket
+   --strict                       will only run commands if the mmctl version matches the server one
```

Comment:
@cwarnermm This section looks broken https://docs.mattermost.com/administration/mmctl-cli-tool.html#mmctl-user-demote

Reply:
Thanks, @amyblais. Fixing via https://github.com/mattermost/docs/pull/4485